t/t0002-basic.t

```diff
@@ -62,6 +62,15 @@ test_expect_success 'pmix.job.size is set correctly on rank 1' '
 		${GETKEY} --proc=* --rank=1 pmix.job.size >size1.out &&
 	test_cmp size.exp size1.out
 '
+test_expect_success 'pmix.univ.size 2 procs is 3' '
```

Comment:
2 procs is **2**

Reply:
Thank you!

I'll fix that typo on my next PR, coming later today I hope.